message_definitions/v1.0/common.xml

```diff
@@ -2218,6 +2218,47 @@
         <param index="6">Empty.</param>
         <param index="7">Empty.</param>
       </entry>
+      <!-- AHRS commands (32000 to 32099) -->
+      <entry value="33000" name="MAV_CMD_AHRS_START" hasLocation="false" isDestination="false">
+        <description>Send command "start data sending" to the AHRS</description>
+        <param index="1">Empty.</param>
```

Comment:
param1 should be an instance number, allowing the user to enable/disable specific backends.